Documentation/building/freebsd-instructions-2.x.md

```diff
@@ -0,0 +1,390 @@
+# FreeBSD 3.X build instructions
```

Comment:
The file name says 2.x, but the title says 3.x.

This should just say FreeBSD build instructions, and we should have just the latest ones. We version the build instructions together with the code. E.g. master has build instructions for master; release/2.0 has the build instructions for release/2.0; etc.

Reply:
These are the instructions to build 2.X. I had a typo. But based on your comment, I'll remove the file.